packages/@react-spectrum/color/src/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/// <reference types="css-module-types" />
```

Comment:
TO DO: Ask what this does

Reply:
It's needed so that running `make build` doesn't print Typescript errors about unresolved CSS imports such as these:
https://github.com/adobe/react-spectrum/blob/2eddd0b274b9f2c418a40ad535aa0164757a76bb/packages/%40react-spectrum/button/src/Button.tsx#L19
If you don't import any CSS in e.g. `HexColorField.ts`, it can be removed.

Reply:
Thanks!